transifex/pull.py

```diff
@@ -317,8 +83,7 @@ def parse_arguments():
     args = parse_arguments()
 
     if args.clone_url:
-        repo = Repo(args.clone_url, skip_compilemessages=args.skip_compilemessages)
-        pull(repo)
+        pull(args.clone_url, skip_compilemessages=args.skip_compilemessages)
     else:
         logger.info('No arguments provided. Using settings.yaml.')
```

Comment:
@rlucioni do we still need to support this? It doesn't seem to work when I run the script like this locally from master. I'd prefer to remove this section if it is not being used by anyone as it makes it more complicated to add options like `--merge-method`, as you would probably want a different one for each repo.